app/commands/submission/representation/trigger_reruns_for_exercise.rb

```diff
@@ -0,0 +1,13 @@
+class Submission::Representation::TriggerRerunsForExercise
+  include Mandate
+
+  queue_as :dribble
+
+  initialize_with :exercise
+
+  def call
+    Submission.where(exercise:).find_each do |submission|
+      Submission::Representation::Init.(submission, type: :exercise, git_sha: exercise.git_sha, run_in_background: true)
```

Comment:
So for the other reruns that we trigged, we set the git sha to the submission's git sha. However, if we did that here, the representer would get the old version of the exercise that does not have the updated representer version.